roseau/load_flow/models/core.py

```diff
@@ -14,18 +14,30 @@
 
 if TYPE_CHECKING:
     from roseau.load_flow.models.buses import Bus
-    from roseau.load_flow.models.lines import Line, Switch
-    from roseau.load_flow.models.transformers import Transformer
+    from roseau.load_flow.models.lines import Line, LineCharacteristics, Switch
+    from roseau.load_flow.models.transformers import Transformer, TransformerCharacteristics
 
 logger = logging.getLogger(__name__)
 
 
+# Only these phases are currently allowed
+Phases = Literal["abc", "abcn"]
+
+
 class Element(ABC):
     """An abstract class to describe an element of an Electrical network"""
 
     def __init__(self, **kwargs):
         self.connected_elements: list[Element] = []
 
+    def _check_phases(self, id: str, **kwargs: str) -> None:
+        # A check on all elements to tell users we only support 3-phase elements for now
+        name, phases = kwargs.popitem()  # phases, phases1 or phases2
+        if phases not in ("abc", "abcn"):
+            msg = f"{type(self).__name__} of id {id!r} got invalid {name} {phases!r}, allowed values are: 'abc', 'abcn'"
+            logger.error(msg)
+            raise RoseauLoadFlowException(msg, RoseauLoadFlowExceptionCode.BAD_PHASE)
```

Comment:
We don't have to limit to only "abc" and "abcn", I think we can already support everyting!